E74 fix translation dbadaptor code ref

tweep opened the pull request:
Hi Will, 

I've edited the VEP.pm and added a line so the references to active db adaptors don't get serialized. 

Cheers, 
    Jan

willmclaren commented:
There's some other stuff in here that I don't think you intended to add; if there are other issues to address (modules/Bio/EnsEMBL/Variation/Pipeline/UpdateVariationFeature.pm ?) then please submit them in a separate pull request.

Also you've made the pull request against release/78; I assume you intended to request against release/74?

tweep commented:
Hi Will,

yes - I intended to submit the pull req against release/74.

Let me check tomorrow what other stuff  I added .... I thought it was a one
line change  but Maybe I added accidentally some other things.

I will look at UpdateVariationFeature.pm as I am not familar with that
module... let's find out what it does....

Thanks,
  Jan


On Wed, Mar 4, 2015 at 1:33 AM, William McLaren <notifications@github.com>
wrote:

> There's some other stuff in here that I don't think you intended to add;
> if there are other issues to address
> (modules/Bio/EnsEMBL/Variation/Pipeline/UpdateVariationFeature.pm ?) then
> please submit them in a separate pull request.
>
> Also you've made the pull request against release/78; I assume you
> intended to request against release/74?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Ensembl/ensembl-variation/pull/12#issuecomment-77123695>
> .
>

willmclaren commented:
Not followed up so closing; please submit a new PR if needed.
